matrix/count_paths.py

```diff
@@ -10,6 +8,13 @@
 #                           matrix. Return count[m-1][n-1]
 # T(n)- O(mn), S(n)- O(mn)
 #
+
+try:
+    raw_input          # Python 2
+except NameError:
+    raw_input = input  # Python 3
```

Comment:
The repo is now Python 3+ so we can replace with `input`. No need for `raw_input`.

Reply:
Infact, there is even no need of `input`. These are just implementations of algorithms. We can hardcode some inputs and add them in tests rather than taking input.